libs/sdk-ui-vis-commons/styles/scss/legend.scss

```diff
@@ -455,6 +489,7 @@ $legend-content-background: var(--gd-chart-backgroundColor, var(--gd-palette-com
     &.horizontal {
```

Comment:
Why were changes to the Heatmap legend necessary in this PR?

Reply:
Because these styles are used also in geo color legend so, therefore, these changes... 😕